### DISCUSSION TITLE
Support for ESP32-S3-BOX-3

### DISCUSSION THREAD

**Discussion by schildbach** [arendst/Tasmota#19471] [Support request]
https://www.espressif.com/en/news/ESP32-S3-BOX-3

It's a nice all-in-one device (including touchscreen and console-like chassis) based on the ESP32-S3. It seems a good fit for smarthome controller projects, gateways/border routers, etc.

The description of the hardware is in this repo: https://github.com/espressif/esp-box/tree/master/hardware

**Comment by Jason2866** [comment]
Tasmota S3 version built qio/opi will run on the device. Just configure the device and the display.ini

**Reply by markbirss** [reply]
Selecting the BOX auto-configuration for this new box-3 device results in a blank display on boot-up

The display reset pin on this new model is inverted ?

`0:00:01.361 APP: Restarting
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0xc (RTC_SW_CPU_RST),boot:0xa (SPI_FAST_FLASH_BOOT)
Saved PC:0x421452aa
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x128
load:0x403c9700,len:0xb90
load:0x403cc700,len:0x28c8
SHA-256 comparison failed:
Calculated: f8f8dc53afbb7f195b56bb611a6f4f83f7d8aebcec67bad654d2c5b3322a2d94
Expected: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
Attempting to boot anyway...
entry 0x403c98b8

00:00:00.001 CMD: Using USB CDC
00:00:00.001 HDW: ESP32-S3 v0.2 (PSRAM)
00:00:00.035 UFS: FlashFS mounted with 12588 kB free
00:00:00.055 CFG: Loaded from File, Count 17
00:00:00.056 FRC: Some settings have been reset (2)
00:00:00.058 I2C: Bus1 using GPIO40(SCL) and GPIO41(SDA)
00:00:00.058 I2C: Bus2 using GPIO18(SCL) and GPIO08(SDA)
00:00:00.062 TFS: File 'mcp23x.dat' not found
00:00:00.108 BRY: Berry initialized, RAM used 5705 bytes
00:00:00.020 DSP: Configured display 'ILI9342'
00:00:00.026 Project tasmota - Tasmota Version 13.3.0.1(lvgl-haspmota)-2_0_14(2023-12-26T13:44:02)
00:00:00.030 LVG: LVGL initialized
00:00:01.782 WIF: Connecting to AP1 Frozen Channel 1 BSSId 54:AF:97:ED:CC:16 in mode 11n as tasmota-5AB2B8-4792...
00:00:03.655 WIF: Connected
00:00:03.912 HTP: Web server active on tasmota-5AB2B8-4792 with IP address 172.16.48.59
16:05:56.887 RSL: INFO1 = {"Info1":{"Module":"ESP32S3-SBOX","Version":"13.3.0.1(lvgl-haspmota)","FallbackTopic":"cmnd/DVES_5AB2B8_fb/","GroupTopic":"cmnd/tasmotas/"}}
16:05:56.889 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-5AB2B8-4792","IPAddress":"172.16.48.59","IP6Global":"","IP6Local":"fe80::3230:f9ff:fe5a:b2b8%st1"}}
16:05:56.890 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":6}}
16:05:56.891 RSL: RESULT = {"POWER":"ON"}
16:05:56.892 RSL: POWER = ON
16:05:59.992 QPC: Reset
16:06:00.943 RSL: STATE = {"Time":"2023-12-26T16:06:00","Uptime":"0T00:00:08","UptimeSec":8,"Heap":220,"SleepMode":"Dynamic","Sleep":10,"LoadAvg":99,"MqttCount":0,"Berry":{"HeapUsed":5,"Objects":56},"POWER":"ON","Wifi":{"AP":1,"SSId":"Frozen","BSSId":"54:AF:97:ED:CC:16","Channel":1,"Mode":"11n","RSSI":56,"Signal":-72,"LinkCount":1,"Downtime":"0T00:00:03"}}
16:06:00.978 RSL: SENSOR = {"Time":"2023-12-26T16:06:00","ANALOG":{"A1":3803}}`

**Reply by sfromis** [reply]
Expect a blank display until you put something on it. However, if it is working, you should briefly see a Tasmota logo shortly after boot.

However, this is **bad**:
`00:00:00.056 FRC: Some settings have been reset (2)`

When Tasmota crashes shortly after boot, it will start to "scale back" by disabling features until able to boot without quickly crashing again. Meaning possible loss of config info. Typical issue is not giving it enough power; do not expect a PC USB port to be a good power source.

**Reply by markbirss** [reply]
Oh ok, thank you for the additional info, but no  logo at all, The Esressif IDF samples all run fine

**Reply by sfromis** [reply]
That does not mean that you can safely assume that you're giving it the power for stable operation of Tasmota. I do not have the specs, but as a starting point guess, I'd want at least 1A of 5V power, which is not available from a standard PC USB port.

**Comment by s-hadinger** [comment]
There is no point in trying HASPmota.tapp if your screen shows nothing. It should display a splash screen at boot. 
If the screen is blank, then look at the template and the display.ini file. 
Please stop reporting this as a HASPmota problem, it is just a screen configuration problem

**Comment by dac2020** [comment]
I deleted my comment here. 
I renamed it so that it didn't say 'broken' and will respond there as I'm not wanting to hijack this persons post.
